src/components/MoneyRequestConfirmationList.js

```diff
@@ -190,19 +194,16 @@ function MoneyRequestConfirmationList(props) {
     const distance = lodashGet(transaction, 'routes.route0.distance', 0);
     const shouldCalculateDistanceAmount = props.isDistanceRequest && props.iouAmount === 0;
 
-    // A flag for verifying that the current report is a sub-report of a workspace chat
-    const isPolicyExpenseChat = useMemo(() => ReportUtils.isPolicyExpenseChat(ReportUtils.getRootParentReport(ReportUtils.getReport(props.reportID))), [props.reportID]);
-
     // A flag for showing the categories field
-    const shouldShowCategories = isPolicyExpenseChat && Permissions.canUseCategories(props.betas) && OptionsListUtils.hasEnabledOptions(_.values(props.policyCategories));
+    const shouldShowCategories = props.isPolicyExpenseChat && Permissions.canUseCategories(props.betas) && OptionsListUtils.hasEnabledOptions(_.values(props.policyCategories));
```

Comment:
Yeah, it is because in some scenarios of split bills there is a case when no `reportID` of `url`, but there is in `IOU`.

The parent component - **MoneyRequestConfirmPage** - has 2 reportID values: 

1. From `url`.
2. From a first participant. It is always a case for split bills.

If I pass the send value as a fallback for the first one, it breaks navigation down, because it does not expect a new `id` during the active flow. 

So, I decided to calculate on a higher level.